trino/sqlalchemy/dialect.py

```diff
@@ -80,49 +81,52 @@ def create_connect_args(self, url: URL) -> Tuple[Sequence[Any], Mapping[str, Any
 
         db_parts = (url.database or "system").split("/")
         if len(db_parts) == 1:
-            kwargs["catalog"] = db_parts[0]
+            kwargs["catalog"] = unquote_plus(db_parts[0])
```

Comment:
some things are `unquote` others `unquote_plus`? Why?

Reply:
Should be unquote_plus everywhere.